simulation-runner/initialize-ssl-from-local-certs.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+die () {
+    echo >&2 "$@"
+    echo "Usage: $0 <ssl_dir> <certname>"
+    echo "   <ssl_dir>: path to a local puppet ssldir"
+    echo "   <certname>: the certname that gatling should use to grab pems for client requests"
+    exit 1
+}
+
+[ "$#" -eq 2 ] || die "2 arguments required, $# provided"
+
+echo "
+
+This script will help you get SSL set up for a gatling puppet simulation.
+It assumes that you have a local puppet environment set up, and will
+pull certs from your ssldir to create a Java keystore file that gatling
+can use during the simulation run.  The keystore file will be created
+in ./target/ssl.
+
+Press enter to continue.
+PRESS ENTER"
+read
+
+SSL_DIR=$1
+CERTNAME=$2
+
+echo "
+
+
+Great.  Now we'll copy the necessary files from the ssldir you specified.
+
+Copying cert files for certname ${CERTNAME} from ${SSL_DIR}."
+
+rm -rf ./target/ssl
+mkdir -p ./target/ssl
+
+cp "${SSL_DIR}/certs/${CERTNAME}.pem" ./target/ssl/hostcert.pem
+cp "${SSL_DIR}/private_keys/${CERTNAME}.pem" ./target/ssl/hostkey.pem
+cp "${SSL_DIR}/certs/ca.pem" ./target/ssl/cacert.pem
+
+echo "Copied files.  Generating keystore file for gatling."
+
+cat ./target/ssl/hostcert.pem ./target/ssl/hostkey.pem > ./target/ssl/keystore.pem
+echo "puppet" | openssl pkcs12 -export -in ./target/ssl/keystore.pem -out ./target/ssl/keystore.p12 -name ${CERTNAME} -passout fd:0
```

Comment:
Might be nice to have the password defined as a local variable somewhere and referenced in all of the commands below rather than having the string "puppet' appear everywhere.